report_qweb_signer/data/defaults.xml

```diff
@@ -2,6 +2,6 @@
 <odoo>
     <record model="ir.config_parameter" id="report_qweb_signer_java_param">
         <field name="key">report_qweb_signer.java_parameters</field>
-        <field name="value">-Xms4M -Xmx4M</field>
+        <field name="value">-Xms4M -Xmx1g -XX:CompressedClassSpaceSize=256m</field>
```

Comment:
Is this change related to the migration ?

Reply:
if i didn't add -XX:CompressedClassSpaceSize=256m to the file, I couldn't run tests (same thing for a gitlab runner)